hummingbot/connector/exchange/huobi/huobi_exchange.pyx

```diff
@@ -599,8 +613,10 @@ cdef class HuobiExchange(ExchangeBase):
                         continue
 
                     execute_amount_diff = s_decimal_0
-                    execute_price = Decimal(data["tradePrice"])
-                    remaining_amount = Decimal(data["remainAmt"])
+                    # tradePrice is documented by Huobi but not sent.
+                    # However, orderprice isn't applicable to all order_types, so we assume tradePrice will be sent for such orders
+                    execute_price = Decimal(data.get("orderPrice", data.get("tradePrice", "0")))
+                    remaining_amount = Decimal(data.get("remainAmt", data["orderSize"]))
```

Comment:
Should we assume there is `tradePrice` first before `orderPrice`? From past interactions, I do remember `tradePrice` being more accurate than `orderPrice`. Also I am assuming that there wouldn't be a case where `execute_price` would be `Decimal("0")` right?

Reply:
After some discussion wth @vic-en, using `order_price` to reflect the `execute_price` will suffice in for `LIMIT` order types.